### PR TITLE
Jroach/add d2l navigation

### PR DESCRIPTION
**Pull request by jeroach**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Brightspace UI is a set of accessible, free, open-source web components for buil
...
 | [d2l-loading-spinner](https://github.com/BrightspaceUI/loading-spinner) | ![Bower version](https://badge.fury.io/bo/d2l-loading-spinner.svg) | [![Build Status](https://travis-ci.org/BrightspaceUI/loading-spinner.svg?branch=master)](https://travis-ci.org/BrightspaceUI/loading-spinner) | ![Polymer 2 supported](https://img.shields.io/badge/Polymer2-supported-blue.svg) |
 | [d2l-menu](https://github.com/BrightspaceUI/menu) | ![Bower version](https://badge.fury.io/bo/d2l-menu.svg) | [![Build Status](https://travis-ci.org/BrightspaceUI/menu.svg?branch=master)](https://travis-ci.org/BrightspaceUI/menu) | ![Polymer 2 supported](https://img.shields.io/badge/Polymer2-supported-blue.svg) |
+| [d2l-navigation](https://github.com/BrightspaceUI/navigation) | ![Bower version](https://badge.fury.io/bo/d2l-navigation.svg) | [![Build Status](https://travis-ci.org/BrightspaceUI/navigation.svg?branch=master)](https://travis-ci.org/BrightspaceUI/navigation) | ![Polymer 2 supported](https://img.shields.io/badge/Polymer2-supported-red.svg) |
```
Comment (jeroach) [inline README.md]: I assume the `red` Polymer2 badge means not supported - is that the right choice? Not sure.

Comment (jeroach) [reply README.md]: Our element is a Polymer 1 hybrid

Comment (svanherk) [reply README.md]: I'm not sure exactly what the colours represent, but I think `blue` works.  `d2l-navigation` is in the same state as the other `blue` elements, maybe we are reserving `green` for full Polymer 2 classes-based elements.

File: brightspace-ui-collection.html [hunk 2]
```diff
@@ -15,6 +15,7 @@
 <link rel="import" href="../d2l-loading-spinner/d2l-loading-spinner.html">
 <link rel="import" href="../d2l-menu/d2l-menu.html">
 <link rel="import" href="../d2l-more-less/d2l-more-less.html">
+<link rel="import" href="../d2l-navigation/d2l-navigation.html">
```
Comment (svanherk) [inline brightspace-ui-collection.html]: Do we want this to be `d2l-navigation-band.html` and then we come back and add the other files as they're created?  I'm not really sure what this file does, I can't find it used anywhere

Comment (jeroach) [reply brightspace-ui-collection.html]: Good point. I updated to use `all-imports.html` just like `d2l-button` does.

Currently that file only imports `d2l-navigation-band` but it saves us from having to update this repo each time we add a new component to `navigation`

**Pull request merged**